Docs/linqsupport.md

```diff
@@ -0,0 +1,234 @@
+LINQ support in Realm Xamarin
+=============================
+
+To make a query with Realm, you use the `Realm.All<T>()` method to get a `IQueryable<T>` instance. 
```

Comment:
All of these should be `cref`'s, but that is probably outside the scope of this PR.